A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -2009,33 +1988,25 @@ class FMCMainDisplay extends BaseAirliners {
         return false;
     }
 
+    /**
+     * VLS for current config (not selected landing config!)
+     */
     getVLS() {
-        const dWeight = SimVar.GetSimVarValue("TOTAL WEIGHT", "kilograms") / 1000;
-        const cg = this.zeroFuelWeightMassCenter;
-        if (((isNaN(cg)) ? 24 : cg) < 25) {
-            switch (true) {
-                case (dWeight <= 50):
-                    return 116;
-                case (dWeight >= 75):
-                    return Math.ceil(139 + .8 * (dWeight - 75));
-                case (dWeight <= 55):
-                    return Math.ceil(116 + .8 * (dWeight - 50));
-                case (dWeight <= 70):
-                    return Math.ceil(120 + dWeight - 55);
-                default:
-                    return Math.ceil(135 + .8 * (dWeight - 70));
-            }
-        }
-        switch (true) {
-            case (dWeight <= 50):
-                return 116;
-            case (dWeight >= 75):
-                return Math.ceil(139 + .8 * (dWeight - 75));
-            case (dWeight <= 55):
-                return Math.ceil(116 + .6 * (dWeight - 50));
-            default:
-                return Math.ceil(119 + dWeight - 55);
+        return SimVar.GetSimVarValue("L:A32NX_SPEEDS_VLS", "Number");
```

Comment:
There is only one consumer left for this. It is some [go-around speed calculation](https://github.com/tracernz/a32nx/blob/97ce2ff93fbffdd42e542bfded32525fe40f0948/A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js#L1288-L1291) which I can't find a reference for in the Airbus docs I have... I'm assuming it wants VLS for current config.